Use system bzip2 library. Get CC and CFLAGS from environment.
 -jer (bug #249140)


--- analog-6.0/src/Makefile.orig	2004-12-19 14:51:30.000000000 +0100
+++ analog-6.0/src/Makefile	2008-12-15 07:34:09.000000000 +0100
@@ -2,16 +2,16 @@
 # Please read docs/Readme.html, or http://www.analog.cx/
 # This is a general Unix-like Makefile: Makefiles for other OS's can be found
 #   in the "build" directory.
-CC = gcc           # which compiler to use: eg cc, acc, gcc. NB Different
+# CC is set in the environment.
 #                    compilers need different CFLAGS, e.g., -O instead of -O2.
 MAKE = make        # which "make" to use
-CFLAGS = -O2    # options, e.g. for optimisation or ANSI compilation.
+# CFLAGS is set in the environment.
 #                 Some OS's need -D_FILE_OFFSET_BITS=64 to support files > 2MB.
 #                 HP/UX cc needs CFLAGS = -Aa (HP/UX 9) or -Ae (HP/UX 10)
 #                 BeOS needs CFLAGS = -O2 -Wl,-L/boot/home/config/lib
 #                 BS2000/OSD needs CFLAGS = -XLLML -XLLMK
 #                 NeXTSTEP needs CFLAGS = -O2 -pipe -no-precomp 
-DEFS =          # any combination of -DNOPIPES -DNODNS -DNODIRENT -DNOGLOB ...
+DEFS = -DHAVE_GD -DHAVE_PCRE -DHAVE_ZLIB -DHAVE_BZLIB
 #                 ... -DNOOPEN -DNOFOLLOW -DNOALARM -DNOGRAPHICS -DNOGMTIME ...
 #                 ... -DEBCDIC -DUSE_PLAIN_SETJMP ...
 #                 ... -DHAVE_GD -DHAVE_ZLIB -DHAVE_BZLIB -DHAVE_PCRE ...
@@ -30,7 +30,7 @@
 #
 OS = UNIX         # Operating system: UNIX, DOS, WIN32, MAC, OS2, OSX, VMS
 #                   RISCOS, BEOS, NEXTSTEP, MPEIX, BS2000, AS400, OS390
-LIBS = -lm  # extra libraries needed; most platforms (but not OS X or BeOS)
+LIBS = -lgd -lz -lbz2 -lpcre -lm -lpng -ljpeg
 #               need -lm LAST
 #             if you defined HAVE_GD above you also need -lgd -lpng -ljpeg -lz
 #             if you defined HAVE_ZLIB above you also need -lz
